pkg/prom/ha/server.go

```diff
@@ -75,6 +76,7 @@ func (c *Config) RegisterFlags(f *flag.FlagSet) {
 func (c *Config) RegisterFlagsWithPrefix(prefix string, f *flag.FlagSet) {
 	f.BoolVar(&c.Enabled, prefix+"enabled", false, "enables the scraping service mode")
 	f.DurationVar(&c.ReshardInterval, prefix+"reshard-interval", time.Minute*1, "how often to manually reshard")
+	f.DurationVar(&c.ReshardTimeout, prefix+"reshard-timeout", time.Second*30, "timeout for cluster-wide reshards and local reshards")
```

Comment:
nitpick: Should we include an extra phrase for "Timeout of 0s indicates no timeout" in the help text here? It's weirdly obvious and not obvious (to me)

Reply:
good point 👍